static/scss/faculty-members.scss

```diff
@@ -0,0 +1,111 @@
+// sass-lint:disable mixins-before-declarations
+.faculty-section {
+  margin: 0 0 20px;
+
+  .container {
+    h2 {
+      margin: 0 0 35px;
+
+      @include media-breakpoint-down(sm) {
+        margin: 0 0 27px;
+      }
+    }
+  }
+}
+
+.faculty-members {
+  padding: 0;
+  margin: 0 -22px;
+  list-style: none;
+  letter-spacing: -0.32em;
+
+  .course-card-description {
+    overflow-y: auto;
+    margin-bottom: 30px;
+  }
+
+  li {
+    @include media-breakpoint-down(md) {
+      width: calc(100% - 44px);
+    }
+
+    letter-spacing: normal;
+    display: inline-block;
+    vertical-align: top;
+    width: calc(50% - 44px);
+    margin: 0 22px 24px;
+
+    .member-card {
+      padding: 24px;
+      background: white;
+      border-bottom: 3px solid transparent;
+      box-shadow: 0 2px 4px 0 rgba(0,0,0,0.16);
+      overflow: hidden;
+
+      .member-info {
+        overflow: hidden;
+        margin-top: 7px;
+
+        @include media-breakpoint-down(md) {
+          margin-top: 20px;
+        }
+
+        h2 {
```

Comment:
Why is this an `h2` and resized to be smaller? Why not just make it an `h4`? The existing size that we have for the `h4` is perfectly good for this